### PR TITLE
Removed deprecated gulp-minify-css in favour of gulp-cssnano

### PR DESCRIPTION
**Pull request by JakeCobley**
gulp-minify-css was deprecated at version 1.2.3

Issue: #1609

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ var cssTasks = function(filename) {
         'opera 12'
       ]
     })
-    .pipe(minifyCss, {
+    .pipe(cssNano, {
       advanced: false,
```
Comment (austinpray) [inline gulpfile.js]: Are these options still relevant?

Comment (JakeCobley) [inline gulpfile.js]: As far as I'm aware advanced: false can be removed. That was an oversight on my behalf.

Comment (austinpray) [inline gulpfile.js]: While you are revising the options: basically disable all "clever" options. Potentially destructive optimizations are not wanted.

**Comment by JakeCobley** [comment]
Updated commit to remove unneeded options.

**Comment by austinpray** [comment]
Looks good.

@landerss0n does this PR work for you on your projects as well?

**Comment by landerss0n** [comment]
Yes it works fine

**Comment by austinpray** [comment]
>While you are revising the options: basically disable all "clever" options. Potentially destructive optimizations are not wanted.

**Comment by JakeCobley** [comment]
Added `safe: true` option, should be sound now.

**Pull request merged**